<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CMapMainWindow</class>
 <widget class="QMainWindow" name="CMapMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>200</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>200</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>500</width>
    <height>250</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>üéØ Radar Display - Window Manager</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow {
    background-color: #ffffff;
}</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="styleSheet">
    <string notr="true">QWidget {
    background-color: #ffffff;
    color: #000000;
    font-family: 'Segoe UI', Arial, sans-serif;
}

QPushButton {
    background-color: #ffffff;
    color: #000000;
    border: 2px solid #000000;
    border-radius: 8px;
    padding: 8px 16px;
    font-weight: 600;
    font-size: 12px;
    min-height: 28px;
}

QPushButton:hover {
    background-color: #f0f0f0;
}

QPushButton:pressed {
    background-color: #e0e0e0;
}

QLabel {
    color: #000000;
    font-weight: 600;
    font-size: 14px;
}</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>12</number>
    </property>
    <property name="leftMargin">
     <number>16</number>
    </property>
    <property name="topMargin">
     <number>16</number>
    </property>
    <property name="rightMargin">
     <number>16</number>
    </property>
    <property name="bottomMargin">
     <number>16</number>
    </property>
    <item>
     <widget class="QLabel" name="titleLabel">
      <property name="text">
       <string>üéØ Dual Monitor Radar Display</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="windowButtonsLayout">
      <property name="spacing">
       <number>8</number>
      </property>
      <item>
       <widget class="QPushButton" name="showPPIButton">
        <property name="text">
         <string>üñ•Ô∏è Show PPI</string>
        </property>
        <property name="toolTip">
         <string>Show PPI Window (F1)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="showControlsButton">
        <property name="text">
         <string>üéõÔ∏è Show Controls</string>
        </property>
        <property name="toolTip">
         <string>Show Controls Window (F2)</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="arrangementButtonsLayout">
      <property name="spacing">
       <number>8</number>
      </property>
      <item>
       <widget class="QPushButton" name="arrangeDualButton">
        <property name="text">
         <string>üìê Dual Monitor</string>
        </property>
        <property name="toolTip">
         <string>Arrange for Dual Monitor (F3)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="fullscreenButton">
        <property name="text">
         <string>üîç Fullscreen</string>
        </property>
        <property name="toolTip">
         <string>Toggle Fullscreen (F4)</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="helpLabel">
      <property name="text">
       <string>üí° Use F1-F4 keys for quick access</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel { color: #000000; font-size: 11px; }</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="styleSheet">
    <string notr="true">QStatusBar {
    background-color: #ffffff;
    color: #000000;
    border-top: 2px solid #000000;
    font-size: 11px;
    font-weight: 500;
    padding: 4px;
}</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>